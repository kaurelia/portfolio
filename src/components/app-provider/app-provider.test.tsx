import { render } from "@testing-library/react";
import AppProvider from "./app-provider";

describe("Test app-provider component", () => {
  it("Check if app-provider renders properly", async () => {
    const { container } = render(<AppProvider />);
    expect(container).toMatchSnapshot();
  });
});
