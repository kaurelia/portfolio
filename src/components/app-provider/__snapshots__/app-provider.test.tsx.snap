// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Test app-provider component Check if app-provider renders properly 1`] = `<div />`;
